system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -135,7 +144,7 @@
 
     .title {
       font: message-box;
-      height: 20px;
+      height: $top-sites-title-height;
```

Comment:
Was this change intentional? The spec shows about 67px from the bottom of the top site square to the top of the Pocket logo, but with this change, there's now about 72px. (I.e., the height was 10px shorter than the line height resulting in 5px closer square.)

before/expected: <img width="110" alt="image" src="https://user-images.githubusercontent.com/438537/29990993-7ea7ea28-8f35-11e7-8258-b81164e5125d.png"> vs <img width="105" alt="image" src="https://user-images.githubusercontent.com/438537/29991024-c1fb0198-8f35-11e7-90db-43c012a54585.png"> :now